odoo/addons/base/tests/test_mail.py

```diff
@@ -86,6 +86,18 @@ def test_html(self):
         for attr in ['javascript']:
             self.assertNotIn(attr, sanitized_html, 'html_sanitize did not remove enough unwanted attributes')
 
+    def test_outlook_mail_sanitize(self):
+        cases = [
+            '<o:p> test 1 </o:p>',
+            '<o:tag1> test 1 </o:tag1>',
+            '<o:> test 1 </o:>',
+        ]
+
+        for content in cases:
+            html = html_sanitize(content)
+            self.assertNotIn('<o:', html)
+            self.assertNotIn('</o:', html)
```

Comment:
Can you test the actual full result in a more realistic context? Having some parent/sibling elements, more similar to the example you give in the commit message. Just to ensure it doesn't mangle everything on accident, since it's a custom regex, not xml tree editing